hubspot3/tickets.py

```diff
@@ -67,47 +80,130 @@ def get(self, ticket_id: str, include_deleted: bool = False, **options) -> Dict:
             "objects/tickets/{}".format(ticket_id), method="GET", **options
         )
 
-    def get_all(self, limit: int = -1, **options) -> list:
+    def _join_output_properties(self, tickets: List[dict]) -> List[dict]:
+        """
+        Join request properties to show only one output per ticketId
+        This will change the first object for each ticketId
+        """
+        joined_tickets_dict = {}
+        for ticket in tickets:
+            ticket_id = ticket["objectId"]
+            if ticket_id not in joined_tickets_dict:
+                joined_tickets_dict[ticket_id] = ticket
+            else:
+                joined_tickets_dict[ticket_id]["properties"] = {**joined_tickets_dict[ticket_id]
```

Comment:
This line (L94-L96) can be written using `dict.update` since you're mutating the base dict:

`joined_tickets_dict[ticket_id]['properties'].update(ticket['properties'])`